packages/editor/src/caches/GLTFCache.ts

```diff
@@ -8,7 +8,7 @@ export default class GLTFCache {
   get(url) {
     const absoluteURL = new URL(url, (window as any).location).href
     if (this.cache.has(absoluteURL)) {
-      return this.cache.get(absoluteURL)
+      return new Map(JSON.parse(JSON.stringify([...this.cache]))).get(absoluteURL)
```

Comment:
Making a whole new map is not great because it will make copy of all assets loaded in cache. But we only need copy of requested object. Also using `JSON` to make copy of asset is not desirable because `JSON` only serialize non functional properties. So all the methods which were in the assets will be gone. So find other way like, `Object.assign` or `clone` method of asset if available (check for deep cloning).